src/components/ui/Button.jsx

```
@@ -0,0 +1,19 @@
+import { PaperPlaneIcon } from "@radix-ui/react-icons";
+import { twMerge } from "tailwind-merge";
+
+const Button = ({ className, children, ...props }) => {
```

ESLint

[error] 'className' is missing in props validation

(react/prop-types)